### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by gaby** [weaveworks/ignite#971]
Hello,

Can someone from weaveworks provide an update regarding the status of `ignite`?

There hasn't been any comments or update since Fall last year. There's several dependency PR's pending too.

Cc @luxas @twelho @darkowlzz @stealthybox @networkop @chanwit

**Comment by haikyuu** [comment]
Quite interested as well. This project is fantastic ✨

**Comment by gaby** [comment]
Same thing is happening to the weaveworks/weave and weaveworks/scope projects. They both havent had releases in over a year

**Comment by gaby** [comment]
@networkop Yeah, seems like the company went full into the GitOps side lf things.

**Comment by jmls** [comment]
so I guess that's a "not moving forward" status then ...

**Comment by woodliu** [comment]
It's a pity if this project beed abandoned, i think it is the best free microVMs manager...

**Comment by gaby** [comment]
> It's a pity if this project beed abandoned, i think it is the best free microVMs manager...

Sadly it was

**Comment by woodliu** [comment]
Hi, i forked this [project](https://github.com/save-abandoned-projects/ignite), and have done some research about ignite and its' depended project [libgitops](https://github.com/weaveworks/libgitops), and now i have created some [issues](https://github.com/save-abandoned-projects/ignite/issues) and [upgrade](https://github.com/save-abandoned-projects/libgitops/pulls?q=is%3Apr+is%3Aclosed) the libgitops.
I hope someone else can help with this wonderful project.
